From 529754f27ff39a6f13f13a35e5d3a8dcd51e4bd0 Mon Sep 17 00:00:00 2001
From: Rom Lemarchand <romlem@google.com>
Date: Fri, 31 May 2013 11:54:35 -0700
Subject: [PATCH 241/634] android: configs: Enable KSM support by default

This will be a noop unless the KSM thread is enabled by userspace

Change-Id: Ia5fde14504cc0da50522e2f875d8d021f9e054ba
Signed-off-by: Rom Lemarchand <romlem@google.com>
---
 android/configs/android-recommended.cfg | 1 +
 1 file changed, 1 insertion(+)

diff --git a/android/configs/android-recommended.cfg b/android/configs/android-recommended.cfg
index 03e0c91..c3eee84 100644
--- a/android/configs/android-recommended.cfg
+++ b/android/configs/android-recommended.cfg
@@ -2,6 +2,7 @@ CONFIG_PANIC_TIMEOUT=5
 CONFIG_KALLSYMS_ALL=y
 CONFIG_PERF_EVENTS=y
 CONFIG_COMPACTION=y
+CONFIG_KSM=y
 # CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
 CONFIG_PM_WAKELOCKS_LIMIT=0
 # CONFIG_PM_WAKELOCKS_GC is not set
-- 
1.9.1

